pkgs/development/python-modules/pylance/default.nix

```diff
@@ -63,6 +64,7 @@ buildPythonPackage rec {
   dependencies = [
     numpy
     pyarrow
+    torch
```

Comment:
`torch` is not a dependency.
Please add it to `optional-dependencies.torch`.
https://github.com/lancedb/lance/blob/508c1a1a856d350f95aeddbd4256366274e7f305/python/pyproject.toml#L3

Reply:
Done. And hopefully done correctly.